[INFO -- 2017-03-07 19:43:37]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-07 19:43:37]
Initialization strings:
[INFO -- 2017-03-07 19:43:38]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-03-07 19:43:38]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-07 19:43:38]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-07 19:43:38]
(4) file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-07 19:43:38]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-07 19:43:38]
(6) set args 228 2048 16 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_a_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_b_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/gold_2048_m-order_228_ths_16_blocks 1
[INFO -- 2017-03-07 19:43:38]
initSignal:28
[INFO -- 2017-03-07 19:43:38]
endSignal:28
[INFO -- 2017-03-07 19:43:38]
seqSignal:15
[INFO -- 2017-03-07 19:43:38]
faultMode:0
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 19:44:10]
pythonException: invalid frame
[INFO -- 2017-03-07 19:44:10]
Backtrace BEGIN:
#0  0x00000000004038fc in L_dgemm_185__par_region0_2_0 () at dgemm.c:224
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffe550 in ?? ()
#3  0x0000000000000000 in ?? ()

219	                        for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
220	                            for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
221	                                CC_arr(i,j) += AA_arr(i,k)*BB_arr(j,k);
222	
223	                    for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
224	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
225	                            C_arr(ig,jg) += CC_arr(i,j);
226	
227	                }
228	            }

Backtrace END
Memory content before bitflip:0x000x000x000x00
Fault Mode: Single bit-flip
base address to flip value: 0x7fffffffde90
address max offset: 4
Memory content after  bitflip:0x000x000x040x00
frame name: L_dgemm_185__par_region0_2_0
symbol name: jj
symbol filename: dgemm.c
symbol line: 183
value: 262144
value address: 0x7fffffffde90
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-03-07 19:44:10]
Fault Injection Successful after 32.746696949s
[INFO -- 2017-03-07 19:44:20]
event type: exit
[EXCEPTION -- 2017-03-07 19:44:20]
exit code: no exit code available
